README.md

```diff
@@ -133,4 +133,6 @@ Additionally, the endpoint PUT /api/users/one/{email}/photo accepts SVG images,
 
 * **AO1 Vertical access controls** - The page /dashboard can be reached despite the rights of user.
 
-* **Broken Function Level Authorization** - The endpoint DELETE `/users/one/:id/photo?isAdmin=` can be used to delete any user's profile photo by enumerating the user IDs and setting the `isAdmin` query parameter to true, as there is no validation of it's value on the server side.
\ No newline at end of file
+* **Broken Function Level Authorization** - The endpoint DELETE `/users/one/:id/photo?isAdmin=` can be used to delete any user's profile photo by enumerating the user IDs and setting the `isAdmin` query parameter to true, as there is no validation of it's value on the server side.
+
+* **IFrame Injection** - The `/testimonials` page takes two URL parameters: `videosrc` and `videotitle` which directly control the src and title attributed of the IFrame at the bottom of this page.
```

Comment:
Please take into account this comment https://brightsec.atlassian.net/browse/APS-99?focusedCommentId=46686 
According to this text,  there is one entry point vulnerable for injecting int to the `src` tag and one another tag.  
But when the engine manages to find vulnerable `src` - it stops the test - there for vector attack of injecting in another tag would not be tested.

Reply:
@alexkutsan I've updated it so we have an IFrame on the main page which is vulnerable through it's title and the IFrame on the testimonials page is vulnerable through the src attribute :)